src/event.js

```diff
@@ -36,7 +36,7 @@ function safeActiveElement() {
 	} catch ( err ) { }
 }
 
-function on( elem, types, selector, data, fn, one ) {
+function on(elem, types, selector, data, fn, one ) {
```

Comment:
format: style mismatch:
`97ea1db9` Replace ⏎⏎ with ␣ in the beginning of the line
`55a8147f` ␣ should be inserted at column 13.